buildifier.sh

```diff
@@ -0,0 +1,118 @@
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#dict-concatenation
+          dict-concatenation
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#duplicated-name
+          duplicated-name
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#filetype
+          filetype
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#git-repository
+          git-repository
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#http-archive
+          http-archive
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#integer-division
+          integer-division
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load
+          load
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#load-on-top
+          load-on-top
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#native-build
+          native-build
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#native-package
+          native-package
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#no-effect
+          
+          # Reason: https://github.com/bazelbuild/buildtools/issues/471
+          # no-effect
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#out-of-order-load
+          out-of-order-load
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#output-group
+          output-group
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#package-name
+          package-name
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#package-on-top
+          package-on-top
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#positional-args
+          positional-args
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#redefined-variable
+          redefined-variable
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#repository-name
+          repository-name
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#same-origin-load
+          same-origin-load
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#string-iteration
+          string-iteration
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#unsorted-dict-items
+          unsorted-dict-items
+
+          # https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#unused-variable
+          unused-variable
+        )
+
+LINT_RULES=$(join , ${LINT_RULES[@]})
+
+echo "Buildifier: (enabled lint rules)"
+echo $LINT_RULES
+echo ""
+
+find . -type f \( -name "*.bzl" -or -name WORKSPACE -or -name BUILD -or -name BUILD.bazel \) ! -path "*/node_modules/*" \
```

Comment:
This is a nice idea, but it actually belongs in the `@bazel/buildifier` package as a new `bin` entry (I imagined `buildifier-all`) and should be implemented in JS so that it can work on Windows too.
But let's do that as a separate task :)
for now can we just list our lint rules in the package.json script to keep this change small?

Reply:
We will then have to list all the lint rules (in the package.json) except the rules we want to temporarily exclude (ie, no-effect). Are we okay with that?

Reply:
fixed: https://github.com/bazelbuild/rules_nodejs/blob/c4b99b9cc66723303d26e4e0d5add8b81e014edd/package.json#L27